modules/freewheel-sspBidAdapter.js

```diff
@@ -0,0 +1,372 @@
+import * as utils from '../src/utils';
+import { registerBidder } from '../src/adapters/bidderFactory';
+// import { config } from 'src/config';

```

Comment:
Can you please remove this commented line if it's no longer needed?

Reply:
removed